substra/sdk/backends/local/compute/worker.py

```diff
@@ -55,19 +55,26 @@ def _get_address_in_container(model_key, volume, container_volume):
 class Worker:
     """ML Worker."""
 
-    def __init__(self):
+    def __init__(self, db: dal.DataAccess):
         self._wdir = os.path.join(os.getcwd(), "local-worker")
-        self._db = db.get()
+        self._db = db
         self._spawner = spawner.get()
 
+    @staticmethod
+    def _is_local(key: str):
+        """Check if the key corresponds to a local
+        or remote asset.
+        """
+        return key.startswith("local_")
```

Comment:
We should use here a constant for "local_".
As this method is duplicated, maybe we could move this method to the DataAccess key? And move inside this class a helper to generate a local key?

Reply:
You're right, DataAccess should contain everything related to local vs remote

The DataAccess class now has 2 functions:
- is_local --> check if the key starts with 'local_'
- get_local_key --> add 'local_' in front of the key

What do you think ?

Reply:
Sounds good, thanks.